users.js

```diff
@@ -156,17 +254,17 @@ function unlock(name, unlocked, noRecurse) {
 		if (!noRecurse) userips = user.ips;
 	}
 	for (var ip in lockedIps) {
-		if (userips && (ip in user.ips) && Users.lockedIps[ip] !== userid) {
-			unlocked = unlock(Users.lockedIps[ip], unlocked, true); // avoid infinite recursion
+		if (userips && (ip in user.ips) && Users.lockedIps[ip][0] !== userid) {
```

Comment:
`Users.lockedIps[ip][0]` isn't this a bit dangerous if it might not be an array?